### DISCUSSION TITLE
Documentação da Sass em Português

### DISCUSSION THREAD

**Discussion by nazarepiedady** [frontendbr/forum#2418] [Mostre seu trabalho]
Olá todos, eu gostaria dar-vos a conhecer o meu trabalho em traduzir a documentação oficial da Sass, a linguagem de superconjunto de CSS, vocês podem encontrar o código-fonte do projeto nesta ligação: https://github.com/nazarepiedady/sass-docs-pt

Dê suas estrelas se gostarem ou quiserem apoiar o projeto de alguma maneira.

**Comment by EdilandoSaturnino** [comment]
Bem útil, vou dar uma lida, amo muito utilizar o Sass, é um ótimo pré-processador.

**Reply by nazarepiedady** [reply]
@EdilandoSaturnino não esquece de partilhar com as pessoas que acreditas que precisam de material como este.

**Reply by EdilandoSaturnino** [reply]
Já compartilhei, eu considero o Sass algo muito útil, principalmente para aqueles que possuem dificuldade em CSS.

**Comment by ghost** [comment]
Obrigada pela contribuição
Isso é algo que estava faltando por aqui

**Reply by nazarepiedady** [reply]
@poluxsandra não esquece de partilhar com as pessoas que acreditas que precisam de material como este.
